airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/test/steps/manifest_only_connectors.py

```diff
@@ -64,7 +64,7 @@ class ManifestOnlyConnectorUnitTests(PytestStep):
 
     title = "Manifest-only unit tests"
     test_directory_name = "unit_tests"
-    common_test_dependencies = ["pytest", "requests-mock"]
+    common_test_dependencies = ["freezegun", "pytest", "pytest-mock", "requests-mock"]
```

Comment:
After talking with max a bit, he informed me that `pyproject.toml` in the test directory doesn't actually do anything for CI tests... These seem like reasonable generic libraries that many of our unit tests uses